codecov.yml

```diff
@@ -75,21 +75,28 @@ flag_management:
 
 # Read more here: https://docs.codecov.com/docs/pull-request-comments
 comment:
+  #### FOR TESTING ####
+  # If you want to test in your PR commenting configurations changes,
+  # comment out this line and uncomment the next line in the next section
+  # require_bundle_changes: false
+  ######################
+  # Post comment if there are changes in bundle size increases greater than value specified
+  require_bundle_changes: 'bundle_increase'
+  ######################
   # This is the addition of carry forward builds and fresh builds, thus, it's the addition
   # of the FE and BE builds
   after_n_builds: 22
-  # Show only the git diff / patch coverage, and exclude any comparison between base and head:
-  layout: 'condensed_header, condensed_files, condensed_footer'
+  # What to render in the comment
+  layout: 'diff, condensed_files'
+  # Do not show the project coverage in the comment
   hide_project_coverage: true
+  show_carryforward_flags: true
```

Comment:
This is new. It's explained [here](https://docs.codecov.com/docs/carryforward-flags#carryforward-flags-in-the-pull-request-comment).

It should show like this:
![image](https://github.com/user-attachments/assets/cf758746-3e80-42ed-903f-9c868d7e9ffa)